source/features/network-info/store.ts

```diff
@@ -85,6 +85,11 @@ export class NetworkInfoStore extends Store {
     const result = await this.networkInfoApi.fetchStatic.execute({});
     if (result) {
       const { cardano } = result;
+      if (cardano.networkName !== environment.CARDANO.NETWORK) {
+        throw new Error(
+          `API network '${cardano.networkName}' does not match environment network '${environment.CARDANO.NETWORK}'`
```

Comment:
```suggestion
          `Cardano GraphQL is connected to" '${cardano.networkName}', whereas the web app is expecting '${environment.CARDANO.NETWORK}'`. The instance of Cardano GraphQL needs to be configured to match our expected environment. 
```

Reply:
Was going to suggest a custom error, but I feel a better approach is to remove the local config and make the app more network agnostic. So a quick error here is fine